nautobot/dcim/templates/dcim/interfaceredundancygroup_retrieve.html

```diff
@@ -0,0 +1,79 @@
+{% extends 'generic/object_retrieve.html' %}
+{% load helpers %}
+
+{% block extra_buttons %}
+{% if perms.dcim.change_interfaceredundancygroup %}
+
+<div class="btn-group">
+    <a href="{% url 'dcim:interfaceredundancygroupassociation_add' %}?group={{ object.pk }}&return_url={% url 'dcim:interfaceredundancygroup' pk=object.pk %}"
+        class="btn btn-primary">
+        <span class="mdi mdi-plus-thick" aria-hidden="true"></span>
+        Add Interface
+    </a>
+</div>
+
+{% endif %}
+{% endblock extra_buttons %}
+
+
+{% block content_left_page %}
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <strong>Interface Redundancy Group</strong>
+    </div>
+    <table class="table table-hover panel-body attr-table">
+        <tr>
```

Comment:
Should we add a `name` row here as well? or `group_id` is sufficient?

Reply:
The name is in the title.  I've renamed `group_id` to `protocol_group_id` to be more explicit.